examples/core/gated_gcn/model.py

```diff
@@ -0,0 +1,58 @@
+"""
```

Comment:
Also I suggest merging `main.py` and `model.py`.

Reply:
yes, I agree